lib/Runtime/Library/InJavascript/Intl.js

```diff
@@ -90,6 +106,33 @@
         MAX: 5
     }, null);
 
+    const _ = {
+        toUpperCase(str) { return callInstanceFunc(StringInstanceToUpperCase, str); },
+        toLowerCase(str) { return callInstanceFunc(StringInstanceToLowerCase, str); },
+        replace(str, pattern, replacement) { return callInstanceFunc(StringInstanceReplace, str, pattern, replacement); },
+        split(str, pattern) { return callInstanceFunc(platform.builtInJavascriptStringEntrySplit, str, pattern); },
+        substring(str, start, end) { return callInstanceFunc(platform.builtInJavascriptStringEntrySubstring, str, start, end); },
+
+        forEach(array, func) { return callInstanceFunc(ArrayInstanceForEach, array, func); },
+        push(array, ...els) { return callInstanceFunc(ArrayInstancePush, array, ...els); },
```

Comment:
>els [](start = 23, length = 3)

nit: els, el, etc -- I think more idiomatic names would be elem/elems

Reply:
Really? I have always used el and els for stuff like this. I can change it, though at the same time these identifiers are never actually used in any particularly meaningful way.